### ISSUE TITLE
Error running first example 

### ISSUE THREAD

**Issue by tatecarson** [antonhornquist/App-sc#1]
When I try to run the first example in the help I get this:

```
ERROR: syntax error, unexpected $end, expecting ')'
  in file 'selected text'
  line 1 char 2:

  ( 
    
-----------------------------------
ERROR: Command line parse failed
-> nil
```

I'm trying to figure out exactly what this is. Is this a model for  class or an executable function?

**Comment by antonhornquist** [comment]
This was broken. I did a new commit with fixes and revised help file so it should work now. But it's a hack so it's guaranteed to be full with bugs. :)

Anyhow, App-sc is me trying out ideas on how to construct mini-apps for SuperCollider with automatic persistance (of both app logic and session state). The idea is to be able to resume apps after a reboot (in between sessions) and they should just work as when you quit SC.

Apps are installed by referring a function to a symbol. Something like this:

```
App.init; // init the framework, can be put in startup.scd.

(
// install an app
App.install(\testApp, { |app|
	/* app logic is in here */
})
)

App.installed; // list of all installed apps

App.run(\testApp); // run the app
App.running; // list of all running apps
App.stop(\testApp); // stop it

// let's say I restart SuperCollider here

App.init; // initialize after SC restart. all installed apps + all stored app state is loaded from Platform.userAppSupportDir +/+ "apps.sctxar" on initialization where it was persisted when SC quit

App.run(\testApp); // app resumes from where it was stopped. no need to reevaluate code above
App.stop(\testApp); // stop the app
App.uninstall(\testApp); // uninstall it and it's removed from memory and will not be persisted to disk on quit
```

Help file has complete examples. In practice this gets a bit messy so I'm not too sure about this anymore. Haven't spent much time on it lately to be honest. If I resume work on this I'll probably aim to make it more declarative.

**Comment by antonhornquist** [comment]
Fixed with 31c19ae4f83709536fb7f938774bdab3205f49ba

**Comment by tatecarson** [comment]
Thanks for sharing, I always learn something from looking through your code.
